addons/hashes/fnc_hashHasKey.sqf

```diff
@@ -23,6 +23,6 @@ Author:
 SCRIPT(hashHasKey);
 
 // -----------------------------------------------------------------------------
-params [["_hash", [], [[]]], "_key"];
+params [["_hash", [] call CBA_fnc_hashCreate, [[]]], "_key"];
```

Comment:
The array is fully evaluated first, so `CBA_fnc_hashCreate` function would be called every time,
even on good input.

Could just use `[[], []]` which will be good enough for this function (`_hash select 1`)